codegen/src/main/scala/akka/grpc/gen/javadsl/Method.scala

```diff
@@ -50,13 +55,24 @@ final case class Method(
 
 object Method {
   def apply(descriptor: MethodDescriptor): Method = {
+
+    // replicates logic from ScalaPB's DescriptorImplicits.MethodDescriptorPimp
```

Comment:
it's quite horrible to have to inline that here and below but since the Java bindings don't have access to ScalaPB's pimp classes, I am not sure how it could be made better?

Reply:
I can try your suggestion indeed. I can't tell what's your final take on it though, should we push for that? Or you want other opinions?

> so some of the logic in that utility might not apply for the Java code
Is there enough java test coverage to rely on CI for that?

Reply:
I don't have a strong preference, so it's up to you as contributor which way to go.

Our test coverage is sufficient that I have confidence that this feature would not break anything, but a comment explaining the situation might be useful to prevent confusion when this code is touched later on.

Reply:
I have added a separate commit to use ScalaPB's `DescriptorImplicits` in a very limited and documented scope of the Java stub generation code